phobos/utils/editing.py

```diff
@@ -306,6 +306,7 @@ def instantiateSubmodel(submodelname, instancename, size=1.0):
     # TODO currently this works only by name binding, we should add links to
     # the group here
     submodelobj['submodel/name'] = submodelname
+    submodelobj['submodelname'] = submodelname
```

Comment:
As stated below, the submodel of Phobos is not to be mistaken for a submodel of the DROCK-plugin, as there is additional information stored within the latter.
Rather adjust the DROCK-plugin to follow the new Phobos convention...